.github/workflows/maven.yml

```diff
@@ -93,12 +93,14 @@ jobs:
 
       - name: Upload code coverage to CodeCov (Main build)
         if: matrix.isMainBuildEnv
-        run: bash <(curl -s https://codecov.io/bash)
+        uses: codecov/codecov-action@v2
```

Comment:
@adamcin Please revert this commit. We have https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/2706/files for that.

Reply:
@kwin reverted